nuxt-frontend/assets/css/global.css

```diff
@@ -86,14 +86,6 @@
     @apply text-[1.25rem] font-bold leading-[1.6rem] tracking-[0.5px] text-[#3d3d3d] md:text-[1.75rem] md:leading-8 lg:text-[2.25rem] lg:leading-[2.729rem];
```

Comment:
change font size with tailwind default classes

Reply:
we are going to remove this classes so no need to change them now